zomes/blog/code/src/main_capability.rs

```diff
@@ -55,35 +53,21 @@ zome_functions! {
     }
 
     get_post: |post_hash: HashString| {
-        // get_entry returns a Result<GetEntryResult, RibosomeError>
-        // It's a RibosomeError if something went wrong
-        // The contents of the GetEntryResult will depend on the options passed
-        // in to get_entry()
-        match hdk::get_entry(post_hash,GetEntryOptions{}) {
+        // get_entry returns a Result<Option<T>, ZomeApiError>
+        // where T is the type that you used to commit the entry, in this case a Blog
+        // It's a ZomeApiError if something went wrong (i.e. wrong type in deserialization)
+        // Otherwise its a Some(T) or a None
+        let result : Result<Option<Post>,ZomeApiError> = hdk::get_entry(post_hash);
```

Comment:
I thought we would have it return just an `Option<T>` and only care about errors in `hdk::get_entry_result`..
But I guess this is ok as well..